backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -0,0 +1,22 @@
+[id="backing-up-applications"]
+= Backing up applications
+include::modules/common-attributes.adoc[]
+:context: backing-up-applications
+
+toc::[]
+
+You back up applications by xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[creating a `Backup` custom resource (CR)]. The `Backup` CR creates backup files for Kubernetes resources and internal images and snapshots for persistent volumes (PVs), if the prerequisites are met.
+
+You can create xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-hooks_backing-up-applications[backup hooks] to run commands before or after the backup operation.
+
+You can xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-scheduling-backups_backing-up-applications[schedule backups] by creating a `Schedule` CR.
+
+You can back up applications on NFS data volumes with xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-applications-restic_backing-up-applications[Restic].
+
+You can back up PVs by using the xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Snapshot Interface (CSI)] if the cloud provider supports CSI and does not have a default Velero plugin.
```

Comment:
>if the cloud provider supports CSI and does not have a default Velero plugin.

Maybe below description is more make sense. 
`if the storage  supports CSI driver and does not have default Velero plugin`, it can include the cloud provider and storage such like OCS 4